### ISSUE TITLE
[css2] CSS 2.2 does not reflect changes to the color type from CSS 3 Color, which is a Rec

### ISSUE THREAD

**Issue by svgeesus** [w3c/csswg-drafts#628]
CSS 2.2 section 4.3.6 Colors
https://www.w3.org/TR/CSS22/syndata.html#value-def-color

This section doesn't seem to reflect the changes made in CSS3 Color, which has been a W3C Recommendation for a while. I thought we were adding pointers from 2.2 to newer work, or indeed deleting sections from 2.2 and replacing them with a pointer to newer work?

**Comment by SelenIT** [comment]
Maybe it would make sense to delete all sections marked as replaced by newer spec [in the "Official Definition" of CSS](https://drafts.csswg.org/css-2015/#css) per CSS-2015 snapshot and replace them with pointers to the newer modules? It would be much easier to read the spec without need to check if the particular section has been superseded by something newer or not.

**Comment by tabatkins** [comment]
That is our general policy, yes - delete if it's entirely superceded, with a pointer to the new spec. (For exactly the reasons you give.)

**Comment by fantasai** [comment]
@tabatkins That is incorrect. Our policy was to *add* a pointer to the new spec, not to remove the section.
